synthesizer/src/vm/verify.rs

```diff
@@ -239,6 +239,7 @@ mod tests {
 
         // Deploy the program.
         let deployment = vm.deploy(&program, rng).unwrap();
+        dbg!(deployment.to_bytes_le().unwrap().len());
```

Comment:
this should probably be removed before merging?

Reply:
Yes, good catch!

```suggestion
```